src/main/webapp/content/scss/global.scss

```diff
@@ -39,19 +39,10 @@ Generic styles
     border: none;
 }
 
-.error {
```

Comment:
I removed some unused css in this file, and I didn't see anything about this.
Should be reverted ?